settings.js

```diff
@@ -22,16 +22,16 @@ function generate(){
     "analysis": {
       "tokenizer": {
         "peliasTokenizer": {
-          "type": "pattern",
-          "pattern": "[\\s,/\\\\-]+"
```

Comment:
The previous pattern matched on `[whitespace, comma, forward/bash slashes and hyphen]`, is this still the case with the `icu_tokenizer`?

Reply:
Good point, I'll try to check if we have relevant tests and if we don't, will write them :+1: